src/java.base/share/classes/java/text/ChoiceFormat.java

```diff
@@ -230,6 +230,12 @@ public class ChoiceFormat extends NumberFormat {
      * Patterns} section. Unlike {@link #setChoices(double[], String[])} this
      * method will throw an {@code IllegalArgumentException} if the {@code
      * limits} are not in ascending order.
+     *
+     * @implNote Given an incorrect pattern, this implementation may either
+     * throw an exception or succeed and discard the incorrect
```

Comment:
I think initially since it was an `implSpec` tag, I didn't want to over-detail the inconsistent behavior as specification. As an `implNote` tag it seems much better to give further explanation. I added details for the two possible exceptions. Let me know if you think I should go into further detail on when a pattern may be discarded if you think that is necessary.